core/src/main/java/org/elasticsearch/common/transport/TransportAddress.java

```diff
@@ -46,5 +46,9 @@
 
     boolean sameHost(TransportAddress other);
 
+    default boolean isLocalAddress() {
```

Comment:
Part of living in a modern Java 8 world is that when you implement an interface you also need to inspect the default methods on the interface?